### PR TITLE
Add support for StatefulSets

### LINKED ISSUE #80: Support for statefulsets, potentially configmaps for discovery
It is common for kstream apps to run as statefulsets. It seems, however, streams-explorer is currently limited to deployment and cronjob-based discovery.

I thus propose to add support for statefulsets and ideally even configmaps for discovery.

### PR DESCRIPTION
**Pull request by disrupted**
Resolves #80 
Depends on #84
- [x] Rebase on main before merging

### CODE REVIEW

File: backend/streams_explorer/core/k8s_app.py [hunk 1]
```diff
@@ -14,21 +17,90 @@
...
+        labels = self.metadata.labels
+        labels_to_use = self.get_labels()
+
+        for key in labels_to_use:
+            value = labels.get(key)
+            if value is not None:
+                self.attributes[key] = value
+            elif self.is_common_streams_app():
+                logger.warning(
+                    f"{self.__class__.__name__} {self.name} does not have a label with the name: {key}"
```
Comment (VictorKuenstler) [inline backend/streams_explorer/core/k8s_app.py:84]: Extract a method to get the name, pls. Could be reused in the following.

File: backend/streams_explorer/core/k8s_app.py [hunk 2]
```diff
@@ -14,21 +17,90 @@
...
+                self.error_topic = env.value
+            elif name == self.get_env_name("EXTRA_INPUT_TOPICS"):
+                self.extra_input_topics = self.parse_extra_topics(env.value)
+            elif name == self.get_env_name("EXTRA_OUTPUT_TOPICS"):
+                self.extra_output_topics = self.parse_extra_topics(env.value)
+
+            if self.name:
+                extractor_container.on_streaming_app_env_parsing(env, self.name)
+
+    def is_common_streams_app(self) -> bool:
```
Comment (VictorKuenstler) [inline backend/streams_explorer/core/k8s_app.py:66]: Rename to `is_streams_bootstrap_app`,  pls

**Review by VictorKuenstler — CHANGES_REQUESTED** [review]

File: backend/streams_explorer/core/k8s_app.py (hunk 2 above)
Comment (disrupted) [reply backend/streams_explorer/core/k8s_app.py]: well spotted ;-)

**Review by VictorKuenstler — APPROVED** [review]
LGTM

**Pull request merged**